pkg/service/test/test.go

```diff
@@ -465,17 +466,39 @@ func (t *tester) testHttp(tc models.TestCase, actualResponse *models.HttpResp) (
 	noise := tc.Noise
 
 	var (
-		bodyNoise   []string
-		headerNoise = map[string]string{}
+		bodyNoise   map[string][]string
+		headerNoise map[string][]string
 	)
 
+	bodyNoise = map[string][]string{}
```

Comment:
you could have initialised them where you have declared them.

Reply:
done